packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -787,7 +792,7 @@ abstract class Element implements BuildContext {
     _parent = parent;
     _slot = newSlot;
     _depth = _parent != null ? _parent.depth + 1 : 1;
-    _active = true;
+    _owner = parent?._owner;
```

Comment:
Not sure how to address this. Previously, if you mounted to a ```null``` parent you still marked the node as active, but now we're assuming that active means a non-null owner. Suggestions?